### PR TITLE
When a user is idle, display the time since they were last active

### PR DESCRIPTION
**Pull request by hnrkndrssn**
I'm sending this through for a peer review of this functionality.

The time displayed is affected by issue #688 as it is displaying difference between local time and UTC time.

idle-since-display

### CODE REVIEW

File: JabbR/Chat.js [hunk 1]
```diff
@@ -144,8 +144,28 @@
...
+        var interval = Math.floor(seconds / 31536000);
+        if (interval > 1) { return interval + "y"; }
+
+        interval = Math.floor(seconds / 2592000);
+        if (interval > 1) { return interval + "mths"; }
+
+        interval = Math.floor(seconds / 86400);
+        if (interval > 1) { return interval + "d"; }
+
+        interval = Math.floor(seconds / 3600); if (interval > 1) { return interval + "h"; }
```
Comment (samandmoore) [inline JabbR/Chat.js]: new line?

File: JabbR/Chat.js [hunk 2]
```diff
@@ -144,8 +144,28 @@
...
+
+        interval = Math.floor(seconds / 60); if (interval > 1) { return interval + "m"; }
```
Comment (samandmoore) [inline JabbR/Chat.js]: ditto

File: JabbR/Chat.js (hunk 1 above)
Comment (hnrkndrssn) [inline JabbR/Chat.js]: yes there should be

File: JabbR/Chat.js (hunk 2 above)
Comment (hnrkndrssn) [inline JabbR/Chat.js]: yep

will update these when I get home tonight

**Comment by samandmoore** [comment]
lgtm after those fixes :)

**Comment by hnrkndrssn** [comment]
Fixed up those two new lines as discussed :)

File: JabbR/Chat.js [hunk 3]
```diff
@@ -144,8 +144,30 @@
         }
     }
 
+    function timeSince(date) {
+        var seconds = Math.floor((new Date(new Date().toUTCString()) - date) / 1000);
```
Comment (davidfowl) [inline JabbR/Chat.js]: Isn't there a plugin that does this :smile:

**Comment by davidfowl** [comment]
Looks good.

File: JabbR/Chat.js (hunk 3 above)
Comment (hnrkndrssn) [inline JabbR/Chat.js]: I googled (with Bing) last night but couldn't really find any, but I'll have another look. I did feel a bit dirty for doing it that way but it was the only way I could find yesterday :)

Comment (kiliman) [inline JabbR/Chat.js]: I use [momentjs](http://momentjs.com) for my projects and it works great.

Comment (samandmoore) [inline JabbR/Chat.js]: :+1: i use moment too. it's really food for this sort of stuff.

Comment (hnrkndrssn) [inline JabbR/Chat.js]: Thanks guys! 

I implemented momentjs into this last night. Just haven't committed it yet :smiley:

**Comment by hnrkndrssn** [comment]
When a user goes Idle
image

**Comment by nschonni** [comment]
What about just having the timestamp when hovering over the idle indicator?

**Comment by davidfowl** [comment]
I like

**Pull request closed without merging**